README.md

```diff
@@ -15,5 +20,14 @@
     };
     ```
 
-2. Now make a pull request.
-3. There is no step 3.
+3. Now make a pull request.
+4. There is no step 4 ;)
+
+## How to build (offline)
+
+1. Install and setup [git](https://git-scm.com/) & [yarn](https://yarnpkg.com/).
+2. Fork and clone: `git clone https://github.com/YOUR_USERNAME/Hack-NASA-using-HTML.git`.
+3. Navigate to project folder: `cd Hack-NASA-using-HTML`.
+4. Install dependencies: `yarn install`.
+5. Run project: `yarn start`.
+6. Your local server will be running at [`https://localhost:8080`](https://localhost:8080).
```

Comment:
It is `http` and not `https`. 
- All browsers are intelligent and will immediately make an `http` call if the other one fails. You can notice a 404 when you open in less intelligent browsers like `Simple Browser` in VSCode.

Reply:
okay 👍🏻 , anything else before another commit?

Reply:
Everything else looks good to me.